Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -21,6 +23,16 @@
 static const int CACHE_VERSION = 13;  // Last changed in PR #3261
 static const int DATASTREAM_VERSION = QDataStream::Qt_5_5;
 
+u64 GetTitleID(const QString& path)
```

Comment:
The title ID should really be read from the volume during `TryReadVolume` like all the other metadata. Things like calling `IsInstalled` is unnecessarily slow otherwise, and reading the title ID during `TryReadVolume` will be needed anyway in the future for handling Wii game caching as well as DolphinWX does.